core/build.gradle.kts

```diff
@@ -1,14 +1,19 @@
 plugins {
-    kotlin("jvm")
+    kotlin("multiplatform")
```

Comment:
Same here. Plugin has nothing to do with multiplatform. If this is a library, then only it needs to be mapped for multiplatform. Otherwise no.

Reply:
Done 👍